### ISSUE TITLE
Seek instead of skip if current track > X min

### ISSUE THREAD

**Issue by frdmn** [beardedspice/beardedspice#117]
Hello,

first of all: great application! It totally replaced the [Media Keys Chrome Extension](https://chrome.google.com/webstore/detail/media-keys-by-swayfm/icckhjgjjompfgoiidainoapgjepncej) for me.

I have one suggestion though: I'm listening to a lot of mixtapes and sets on SoundCloud that are usually longer than 1 hous (up to 3 or something). I would love to have a feature that seeks for example the next 5 minutes, just to skip the current track on the mix instead of skipping the whole set, if the current audio's duration is longer than 60 minutes.

I'm not sure if thats possible, but it would be really nice!

Thanks

**Comment by Stillness-2** [comment]
You describe very special case. What if track size 59 min or 58 min? I think you will want skip only 5 min for these tracks too. So where's the line when we need to move to the algorithm "5 minutes"? :) 
Right approach, this is to make separate shortcuts for seeking on track. But this is another story, which requires some time. :) At this moment i can suggest to press "Raise Active Player Window" shortcut, to seek on track, to press "Raise Active Player Window" again, that be to return to work. :)

**Comment by frdmn** [comment]
@Stillness-2 I agree that this is probably a bit special. The seek shortcut would be a nice alternative. I'll be patient then :)
